data/brands/shop/bicycle.json

```diff
@@ -82,6 +82,16 @@
         "shop": "bicycle"
       }
     },
+    {
+      "displayName": "Peleton",
+      "id": "peleton-c50902",
+      "locationSet": {"include": ["ca"]},
+      "tags": {
+        "brand": "Peleton",
+        "name": "Peleton",
+        "shop": "bicycle"
+      }
+    },
```

Comment:
There's already an entry for Peloton in `shop/sports` (https://nsi.guide/?id=peloton-18eda3) probably because they also sell treadmills and other fitness related products besides bikes. I've added the `matchName` to the existing entry and added shop=bicycle as a `matchTag` so that any locations tagged as that also get matched.